srcpkgs/xdg-desktop-portal-wlr/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'xdg-desktop-portal-wlr'
+pkgname=xdg-desktop-portal-wlr
+version=0.1.0
+revision=1
+build_style=meson
+configure_args="-Dsystemd=disabled"
+hostmakedepends="pkg-config"
+makedepends="elogind-devel pipewire-devel wayland-devel wayland-protocols"
+short_desc="xdg-desktop-portal backend for wlroots"
```

Comment:
`short_desc` is `Title case` not `lower case`

Why does travis not run?

Reply:
Since `xdg-desktop-portal` is the program name, you can try to write like the KDE one: `short_desc="Backend implementation for xdg-desktop-portal that is using Qt/KF5"`

Reply:
Anyway, is this correct `short_desc`, other `xdg-desktop-portal-*` has:

> Backend for xdg-desktop-portal implemented by ...

This one is the reverse??

Reply:
Another way to make xlint happy is triple quoting. short_desc="""xdg ...